src/Microsoft.ML.Data/Scorers/FeatureContributionCalculation.cs

```diff
@@ -15,39 +15,26 @@
 using Microsoft.ML.Runtime.Model;
 using Microsoft.ML.Runtime.Numeric;
 
-[assembly: LoadableClass(typeof(IDataScorerTransform), typeof(FeatureContributionCalculationTransform), typeof(FeatureContributionCalculationTransform.Arguments),
-    typeof(SignatureDataScorer), "Feature Contribution Transform", "fct", "FeatureContributionCalculationTransform", MetadataUtils.Const.ScoreColumnKind.FeatureContribution)]
+[assembly: LoadableClass(typeof(IDataScorerTransform), typeof(FeatureContributionScorer), typeof(FeatureContributionScorer.Arguments),
+    typeof(SignatureDataScorer), "Feature Contribution Scorer", "fcc", "FeatureContributionCalculationScorer", MetadataUtils.Const.ScoreColumnKind.FeatureContribution)]
 
-[assembly: LoadableClass(typeof(ISchemaBindableMapper), typeof(FeatureContributionCalculationTransform), typeof(FeatureContributionCalculationTransform.Arguments),
-    typeof(SignatureBindableMapper), "Feature Contribution Mapper", "fct", MetadataUtils.Const.ScoreColumnKind.FeatureContribution)]
+[assembly: LoadableClass(typeof(ISchemaBindableMapper), typeof(FeatureContributionScorer), typeof(FeatureContributionScorer.Arguments),
+    typeof(SignatureBindableMapper), "Feature Contribution Mapper", "fcc", MetadataUtils.Const.ScoreColumnKind.FeatureContribution)]
```

Comment:
>fcc [](start = 69, length = 3)

the side effect is that the old models that use FCC are no longer loadable. Is this intended?

Reply:
Is it? We didn't change the model loading signatures, I hope, only the user facing command line invocation?

---
In reply to: [243093869](https://github.com/dotnet/machinelearning/pull/1847#discussion_r243093869) [](ancestors = 243093869)

Reply:
I can keep fct and wtf as command line invocation so that the old commands keep working? 

---
In reply to: [243102401](https://github.com/dotnet/machinelearning/pull/1847#discussion_r243102401) [](ancestors = 243102401,243093869)